cmd/save_test.go

```diff
@@ -926,6 +925,122 @@ func TestSaveTwiceWithTransform(t *testing.T) {
 	}
 }
 
+func TestSaveTransformUsingPrev(t *testing.T) {
```

Comment:
grumble grumble grumble tests in lib. Then again these are integration tests. I take back the grumbling.